test/cypress/integration/widgets/deployments_spec.js

```diff
@@ -0,0 +1,205 @@
+describe('Deployments widget', () => {
+    const blueprintName = 'deployments_test_hw';
+    const deploymentName = 'deployments_test_hw_dep';
+    const siteName = 'Zakopane';
+    const site = { name: siteName };
+    const blueprintUrl =
+        'https://github.com/cloudify-community/blueprint-examples/releases/download/latest/simple-hello-world-example.zip';
+
+    const searchForDeployment = name =>
+        cy.get('.deploymentsWidget').within(() => {
+            cy.get('.input input')
+                .clear()
+                .type(name);
+            cy.get('.input.loading').should('be.not.visible');
+        });
+
+    const actOnDeployment = (name, action) => {
+        searchForDeployment(name);
+        cy.contains('div.row', name).within(() => cy.get('.menuAction').click());
+        cy.get('.popupMenu > .menu').within(() => cy.contains(action).click());
```

Comment:
Is this pattern (`within` with a single statement) intentional? I think it can be always expressed in a simpler way, for example:
```suggestion
        cy.contains('div.row', name).find('.menuAction').click();
        cy.get('.popupMenu > .menu').contains(action).click();
```
This comment applies to other places in the same file as well.

Reply:
No, not intentional. Good comment, fixed it.